Pre-migration CQA task for the REST API title (#4937)
theforeman/foreman-documentation: guides/common/modules/con_api-syntax.adoc

= API syntax

[role="_abstract"]

> Understanding the basic syntax of API requests and JSON responses enables you to construct valid API calls and interpret the returned data.

Reviewer: "X enables you" is the key word for feature-centric language. If you can, please rephrase it.

Sorry. I must have missed it in the first review.

Reviewer: ```suggestion
Understand the basic syntax of API requests and JSON responses so you can construct valid API calls and interpret the returned data.
```
